cargo-espflash/README.md

```diff
@@ -93,6 +93,21 @@ It is _not_ currently possible to use `cargo-espflash` from within WSL1. There a
 
 It is also _not_ possible to flash chips using the built-in `USB_SERIAL_JTAG` peripheral when using WSL2, because resetting also resets `USB_SERIAL_JTAG` peripheral, which then disconnects the chip from WSL2. Chips _can_ be flashed via UART using WSL2, however.
 
+To be able to flash within WSL2, `systemd` should be enabled. To do so, create or edit `/etc/wsl.conf` and add the following:
```

Comment:
```suggestion
To be able to flash within WSL2, [`systemd` should be enabled](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#systemd-support). To do so, create or edit `/etc/wsl.conf` and add the following:
```

I think it's important to link the Windows documentation of how to do it. Also, they mention `sudo` might be required.

We could also mention that we need to restart the WSL instance(s), hence the shutdown command from bellow.

Reply:
Good idea!